helper-cli/src/main/kotlin/commands/packagecuration/PackageCurationCommand.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.packagecuration
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+
+internal class PackageCurationCommand : CliktCommand() {
+    init {
+        subcommands(
+            SplitCurationsCommand()
```

Comment:
The command line call would be `orth package-curation split-curations`.
What do you think about just `orth package-curation split`?

(maybe this slightly gives a hint why introducing the grouping that early is difficult. E.g. does the subcommand name `package-curation` fit further future subcommands?)

Reply:
> What do you think about just `orth package-curation split`?

Yes. I had this thought as well: https://github.com/oss-review-toolkit/ort/pull/4157#issuecomment-859522680

> E.g. does the subcommand name `package-curation` fit further future subcommands?

I think it does, but I also tried to name it a little more generic by using `curation-helper`, which is somehow duplicated wording for ORT's `helper-cli`.